release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2486,4 +2486,45 @@ link:https://access.redhat.com/solutions/6078781[{product-title} 4.7.13 containe
\ No newline at end of file
+To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-16"]
+=== RHSA-2021:2286 - {product-title} 4.7.16 bug fix and security update
+
+Issued: 2021-06-15
+
+{product-title} release 4.7.16, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2021:2286[RHSA-2021:2286] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2287[RHBA-2021:2287] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6115681[{product-title} 4.7.16 container image list]
+
+[id="ocp-4-7-16-features"]
+==== Features
+
+[id="ocp-4-7-16-virt-platform-enhancement"]
+===== virt_platform enhancement
+
+With this update, users can now collect the `virt_platform` metric. The `virt_platform` metric is needed for the Insights' rules to determine the virtual platform of a cluster. This information is stored in the IO archive of the `config/metric` file.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960645[*BZ#1960645*])
+
+[id="ocp-4-7-16-bug-fixes"]
+==== Bug fixes
+
+* Previously, use of a CoreDNS plugin did not allow queries to be forwarded if they were not answered by the local server. Consequently, some queries for DNS name in the cluster domain would incorrectly fail. This update changes to a CoreDNS plugin that correctly forwards queries, which allows all valid queries in the cluster domain to work correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1962288[*BZ#1962288*])
+
+* During a previous bug fix, the download link for pod logs was changed to a standard HTML anchor element with an empty download attribute. Consequently, the download file lost the default file name format. This update adds a file name to the anchor element download attribute, so that a default file name, formatted as `<pod-name>-<container-name>.log`, is used when downloading pod logs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951210[*BZ#1951210*])
+
+* Previously, the vsphere-problem-detector, newly introduced in {product-title} 4.7, required valid vSphere credentials to operate successfully. Consequently, {product-title} clusters without valid vSphere credentials were marked as `Degraded`. With this update, vsphere-problem-detector does not mark the cluster as `Degraded`, and instead raises an alert so that they will continue to run.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1959546[*BZ#1959546*])
```

Comment:
s/vsphere-problem-detector/`vsphere-problem-detector`
Probably? Might be worth finding how this is referred to already elsewhere - it might have a real name, but if not it probably should be in backticks.